tron/commands/authentication.py

```diff
@@ -0,0 +1,70 @@
+import os
+from typing import cast
+
+from botocore.credentials import InstanceMetadataFetcher
+from botocore.credentials import InstanceMetadataProvider
+
+from tron.commands.cmd_utils import get_client_config
+
+try:
+    from vault_tools.paasta_secret import get_client as get_vault_client  # type: ignore
```

Comment:
hmm, i assume this is 'cause we don't have a py.typed marker in vault_tools and thus mypy isn't parsing the types?

Reply:
or because `hvac` is not typed?

Reply:
(might be worth commenting the `# type: ignore`s with the reasoning behind them - e.g., `# type: ignore  # foo 'cause bar`)